deployment/environment/devenv/don.go

```diff
@@ -75,22 +76,25 @@ func (don *DON) NodeIds() []string {
 }
 
 func (don *DON) CreateSupportedChains(ctx context.Context, chains []ChainConfig, jd JobDistributor) error {
-	var err error
+	g := new(errgroup.Group)
 	for i := range don.Nodes {
-		node := &don.Nodes[i]
-		var jdChains []JDChainConfigInput
-		for _, chain := range chains {
-			jdChains = append(jdChains, JDChainConfigInput{
-				ChainID:   chain.ChainID,
-				ChainType: chain.ChainType,
-			})
-		}
-		if err1 := node.CreateCCIPOCRSupportedChains(ctx, jdChains, jd); err1 != nil {
-			err = multierror.Append(err, err1)
-		}
-		don.Nodes[i] = *node
+		g.Go(func() error {
+			node := &don.Nodes[i]
```

Comment:
same here, `i `can get shadowed , should tie i to local scope
```suggestion
                 i:=i
		g.Go(func() error {
			node := &don.Nodes[i]
```